app/views/components/_published-dates.html.erb

```diff
@@ -6,7 +6,7 @@
   link_to_history ||= false
 %>
 <% if published || last_updated %>
-<div class="app-c-published-dates" <% if history.any? %>id="history" data-module="toggle"<% end %>>
+<div <% if history.any? %>id="history" data-module="toggle" class="app-c-published-dates__history" <% else %> class="app-c-published-dates" <% end %>>
```

Comment:
This isn't quite how BEM works:

`app-c-published-dates__history` should represent a history block within `app-c-published-dates`.

`app-c-published-dates` should always be present

Then if the "history" version is different it can be modified:

`app-c-published-dates app-c-published-dates--history`

Reply:
Ok, I can review and fix this.